samples/quarkus/src/main/java/io/javaoperatorsdk/webhook/admission/sample/quarkus/AdmissionControllerConfig.java

```diff
@@ -42,7 +42,9 @@ public AdmissionController<Pod> mutatingController() {
   @Named(VALIDATING_CONTROLLER)
   public AdmissionController<Pod> validatingController() {
     return new AdmissionController<>((resource, operation) -> {
-      if (resource.getMetadata().getLabels().get(APP_NAME_LABEL_KEY) == null) {
+      if (resource.getMetadata() == null
```

Comment:
The metadata actually never can be null. Maybe this check.

Reply:
Yes, that was too much. Removed.